### DISCUSSION TITLE
Simulation of the YRC1000 and the Motologix interface

### DISCUSSION THREAD

**Discussion by olsenab** [YaskawaEurope/mlx-examples#40] [Q&A]
We are currently operating a GP 400 on a YRC-1000 with Motologix on a Siemens 1500 CPU. I have successfully installed the Motologix library using TIA (Siemens PLC programming tool).

My current challenge lies in simulating the YRC1000 with the Motologix interface towards a PLC. I am seeking a solution similar to Motosim EG-VRC, which can communicate with a PLC for testing purposes before deployment on the hardware.

What is my option?

**Comment by yeu-degroot** [comment]
Today, there is still "hardware in the loop" required for simulating a MotoLogix robot system. The only part which can be simulated is the manipulator:

![image](https://github.com/YaskawaEurope/mlx-examples/assets/52652923/68bd5647-7797-4b5b-bd75-3db499ff1150)


**We are currently developing the MotoLogix function for MotosimEG-VRC**, where there will no longer be "hardware in the loop". Release is expected in the second half of 2024.

**Reply by GME-JR** [reply]
Hi,

For an upcoming project we are also looking to simulate a YRC1000 with Motologix interface, without using hardware for now.
Is this feature still expected to launch sometime these coming months?

**Reply by yeu-degroot** [reply]
Sadly no. I got informed that the global release will most probably shift into early 2025.

**Reply by GME-JR** [reply]
Hi, I haven't been able to find any new info about this 'software only' simulation. Any updates regarding release?

**Reply by yeu-degroot** [reply]
Slightly delayed, I expect a release towards the summer (Q2).
